requirements/base.in

```diff
@@ -1,5 +1,4 @@
 asana
-black~=23.0
```

Comment:
Again, this does basically nothing until we upgrade dc_django_utils. The version we're on pulls in a lot of linting/code style packages we don't really need anymore

Reply:
resolved https://github.com/DemocracyClub/UK-Polling-Stations/pull/8082/commits/defa89260b094e6e6295fa0acd61702b079754da